### ISSUE TITLE
Docstring for `Palette` displays unsupported `name` argument

### ISSUE THREAD

**Issue by mezarque** [aleferna12/colorir#3]
Thanks for building this color management tool - it has a lot of useful features.

I think there's an issue with the docstring for the `Palette` class. It states that there's a `name` argument that the class will accept, but neither `Palette` nor `PaletteBase` seem to use a `name` argument in their `__init__`.

```python
class Palette(PaletteBase):
    """Class that holds colors values associated with names.

    Examples:
        >>> palette = Palette(red="#ff0000") # Uses default color format
        >>> palette.red
        Hex('#ff0000')

        For more examples see the documentation of the :mod:`~colorir.palette` module.

    Args:
        name: Name of the palette which will be used to save it with :meth:`Palette.save()`.
        color_format: Color format specifying how the colors of this :class:`Palette` should be
            stored. Defaults to the value specified in
            :const:`config.DEFAULT_COLOR_FORMAT <colorir.config.DEFAULT_COLOR_FORMAT>`.
        colors: Colors that will be stored in this palette.
    """
```

When I try to use the `name` argument, as below, I receive an error, possibly due to the parsing of `color_kwargs` in the `__init__` of the `Palette` class.

```python
import colorir as cir

_ALL_COLORS = {
    # Core colors
    "lightgrey": "#EBEDE8",
    "shell": "#EDE0D6",
    "dawn": "#F8F4F1",
    "seafoam": "#F9FCF0",
    "tangerine": "#FFB984",
    "black": "#09090A",
    "charcoal": "#484B50",
    "marineblue": "#8A99AD",
    "forest": "#596F74",
}

cir.Palette(colors = _ALL_COLORS, name = "All")
```

```
ValueError: colors can be passed either through the 'colors' parameter or through kwargs but not both
```

It would be nice to have a `name` property of the `Palette` class that's specified at object creation, rather than just when saving the Palette. It seems that this was potentially included in the past, but might have been removed.

**Comment by aleferna12** [comment]
Hey, thanks for noticing that! I'll fix it next minor release when i have some time.

Yeah specifying name on object creation was deprecated because the name was used only to save the palette anyway, so moving it to be an argument of the save method made more sense. Especially because otherwise "name" becomes a required argument for the constructor and that makes creating a palette cumbersome.

**Comment by aleferna12** [comment]
Closed as part of the latest release (2.0.5)
